cluster/config-defaults.yaml

```diff
@@ -107,9 +107,10 @@ scalyr_team_token: ""
 
 prometheus_cpu: "1000m"
 prometheus_mem: "4Gi"
-prometheus_mem_min: "1Gi"
+prometheus_mem_min: "2Gi"
```

Comment:
What's the motivation for increasing the min? It seems like this should work fine from your description?

> The smallest are less than a GB in size

Reply:
The min size of the memory determines how large the TSDB can be. This means that for large clusters `1Gi` might be too small resulting in only a few hours of retention. We can keep it at `1Gi` and then customize this on a cluster-by-cluster basis but it's just more work.

Reply:
Was there a metric we can use to determine the retention time for different clusters? This way we can evaluate if it later makes sense to lower it.

If I look over the last hour only 10% of Prometheis are using more than `2Gi` memory.

![image](https://user-images.githubusercontent.com/128566/74915247-310f4080-53c4-11ea-9e17-d01549e500e4.png)

Reply:
We're now tracking the retention time for each Prometheus, to understand how this change impacts it!

Reply:
![image](https://user-images.githubusercontent.com/128566/74939773-bc063000-53f0-11ea-8136-734d8b493d86.png)